command/server.go

```diff
@@ -372,10 +376,33 @@ func (c *ServerCommand) AutocompleteFlags() complete.Flags {
 }
 
 func (c *ServerCommand) parseConfig() (*server.Config, error) {
-	// Load the configuration
+	if len(c.flagConfigs) == 0 {
+		return server.NewConfig(), nil
+	}
+
+	var kmses []*configutil.KMS
+
+	for _, path := range c.flagConfigs {
+		ks, err := configutil.LoadConfigKMSes(path)
+		if err != nil {
+			return nil, fmt.Errorf("error checking for config KMS block at path %s: %w", path, err)
+		}
+		kmses = append(kmses, ks...)
+	}
+
+	for _, kms := range kmses {
+		if strutil.StrListContains(kms.Purpose, "config") {
+			if c.configKMS != nil {
+				return nil, errors.New("only one seal/kms block marked for \"config\" purpose is allowed")
+			}
+			c.configKMS = kms
+		}
+	}
+
```

Comment:
Nitpick: Could refactor for code reuse w/ the operator encrypt/decrypt, but nbd.